tensorrt_llm/_torch/modules/mamba/ssd_combined.py

```
@@ -215,38 +230,42 @@ def mamba_chunk_scan_combined(
-        dt_bias=dt_bias,
-        initial_states=initial_states,
-        seq_idx=seq_idx,
-        chunk_indices=chunk_indices,
-        chunk_offsets=chunk_offsets,
-        cu_seqlens=cu_seqlens,
-        dt_softplus=dt_softplus,
-        dt_limit=dt_limit,
-        mamba_ssm_cache_dtype=mamba_ssm_cache_dtype)
+        assert (cu_seqlens is not None
+                ), "cu_seqlens must be provided if return_varlen_states is True"
+    out_x, dt_out, dA_cumsum, states, final_states, *rest = (
```

Ruff (0.14.11)

Unpacked variable `out_x` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)